src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/CSharpVirtualDocument.cs

```diff
@@ -84,5 +88,17 @@ public override VirtualDocumentSnapshot Update(IReadOnlyList<TextChange> changes
         }
 
         private CSharpVirtualDocumentSnapshot UpdateSnapshot() => new CSharpVirtualDocumentSnapshot(Uri, TextBuffer.CurrentSnapshot, HostDocumentSyncVersion);
+
+        private void MakeEmptyEdit()
```

Comment:
Throw this into TextBufferExtensions so the C#/HTML documents use the same code path.